metadata-ingestion/tests/integration/dbt/dbt_to_file.yml

```diff
@@ -0,0 +1,11 @@
+---
+source:
+  type: "dbt"
+  config:
+    manifest_path: "./dbt_manifest.json"
+    catalog_path: "./dbt_catalog.json"
```

Comment:
since the integration test is using Pipeline.create, I don't think this file is actually being used

Reply:
yah, I originally used this file before I realized the test harness made it unworkable... Then forgot to nuke it, thanks and appologies.